### DISCUSSION TITLE
OK to delete source?

### DISCUSSION THREAD

**Discussion by trevorhardy** [coin-or/coinbrew#94] [Q&A]
For one of my projects I'm trying to build up a small (disk space) Docker image starting from Alpine which is leading me to build CBC locally (since there's no APK package). After the build and install, I can delete all the cloned in repos, right? That is, does the install step moves all the build product out of the repo folders?

**Comment by tkralphs** [comment]
Yes, you can delete all the cloned repos after installation.
